specification/DigitalOcean-public.v2.yaml

```
@@ -265,6 +265,13 @@ tags:
 
   - name: Floating IP Actions
     description: >-
```

```suggestion
    description: |
```

Change this will stop it from eating the new lines.